### PR TITLE
Test properties of triangle normal.

### PR DESCRIPTION
**Pull request by jeschkies**

### CODE REVIEW

File: lib/triangle.h [hunk 1]
```diff
@@ -31,12 +31,13 @@ class Triangle {
...
         , u(vertices[1] - vertices[0])
         , v(vertices[2] - vertices[0])
-        , normal(u^v)
         , uv(u*v)
         , vv(v*v)
         , uu(u*u)
         , denom(uv * uv - uu * vv)
-    {}
+    {
+        normal = (u^v).Normalize();
```
Comment (boxdot) [inline lib/triangle.h]: You can do it directly in the initializer, since .Normalize() also returns the vector.

File: tests/test_triangle.cpp [hunk 2]
```diff
@@ -0,0 +1,25 @@
+#include "helper.h"
+#include "../lib/triangle.h"
+
+#include <catch.hpp>
+
+TEST_CASE("Test Triangle normal", "[triangle]")
```
Comment (boxdot) [inline tests/test_triangle.cpp]: Nice.

File: lib/triangle.h (hunk 1 above)
Comment (jeschkies) [inline lib/triangle.h]: But what if `Normalize` throws an exception?

**Comment by jeschkies** [comment]
Radiosity test broke. :/

**Comment by jeschkies** [comment]
Okay, radiosity is broken.

**Pull request merged**